input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+        </tr>
+        <tr>
+            <td style="width: 33.33%;">Observation.performer minimum cardinalities differ: IPS requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" style="width: 33.33%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Observation-results-pathology-uv-ips.html">Observation Results: pathology (IPS)</a></td>
+            <td style="width: 33.33%;">Observation.status fixed value differs: IPS requires fixed value 'final' vs AU Core allows multiple values including 'final'.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;">Observation.subject.reference element minimum cardinalities differ: IPS requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;">Observation.performer minimum cardinalities differ: IPS requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 33.33%;"><a href="StructureDefinition-au-core-procedure.html">AU Core Procedure</a></td>
+            <td rowspan="2" style="width: 33.33%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Procedure-uv-ips.html">Procedure (IPS)</a></td>
+            <td style="width: 33.33%;">Procedure.subject.reference minimum cardinalities differ: IPS requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;">Procedure.performed[x] minimum cardinalities differ: IPS requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" style="width: 33.33%;"><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
+            <td rowspan="3" style="width: 33.33%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Observation-tobaccouse-uv-ips.html">Observation - SH: tobacco use</a></td>
+            <td style="width: 33.33%;">Observation.subject.reference minimum cardinalities differ: IPS requires minimum = 1 vs AU Core minimum = 0.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;">Observation.code.coding differs: IPS requires LOINC 72166-2 which is not included in US Core.</td>
+        </tr>
+        <tr>
+            <td style="width: 33.33%;">Observationvalue[x].valueCodeableConcept binding differs: Required binding to <a href="https://hl7.org/fhir/uv/ips/STU1.1/ValueSet-current-smoking-status-uv-ips.html">Current Smoking Status - IPS</a> in IPS vs extensible binding to <a href="https://healthterminologies.gov.au/fhir/ValueSet/smoking-status-1">Smoking Status</a> in AU Core.</td>
         </tr>
     </tbody>
-</table>
\ No newline at end of file
+</table>
+
+##### Additional profiles 
+
+AU Core has no equivalent for the following profiles defined in the International Patient Summary: 
```

Comment:
Suggest reword this sentence to be "This version of AU Core has no equivalent profile for the following IPS profiles:"

Reply:
Done